### PR TITLE
 fix toolbar dropdown button border

### PR DESCRIPTION
**Pull request by dimbslmh**
border of dropdown not correct:
image

fix:
image

### CODE REVIEW

**Comment by dimbslmh** [comment]
fix dropdown button alignment:
image


fix:
image

File: src/themes/bulma/bootstrap-table-bulma.scss [hunk 1]
```diff
@@ -25,17 +25,22 @@
       margin-right: 0;
     }
 
-    .keep-open.button.dropdown {
+    .button.dropdown {
       padding: 0;
-      margin: 0;
       border: 0;
-
+
```
Comment (wenzhixin) [inline src/themes/bulma/bootstrap-table-bulma.scss]: Should remove the spaces here.

Comment (dimbslmh) [reply src/themes/bulma/bootstrap-table-bulma.scss]: Done. https://github.com/wenzhixin/bootstrap-table/pull/4236/commits/64786dc59930a5e8e069a576b611b6ce1545d99b

**Comment by wenzhixin** [comment]
Thanks for your contribution!

**Pull request merged**